### ISSUE TITLE
Importing movies but all are missing

### ISSUE THREAD

**Issue by ghost** [Radarr/Radarr#1705]
So, I am trying to import movies (That were all downloaded by radarr) to a radarr on a different PC.  The issue is that it is reconizing the movie but it is showed as "missing" even when scanning it multiple times.  The only thing I can think of is that I added a "- [{Quality Title}]" to the end of the movie? Is that not aloud? The folder structure has been unchanged.

**Comment by galli-leo** [comment]
That should be fine. Is it only one movie or many?

**Comment by jonesb63** [comment]
All movies.  I also made sure to select a quality profile

**Comment by galli-leo** [comment]
Are the movies Already in the Radarr on the other pc?

**Comment by jonesb63** [comment]
Yes, but it is an identical copy of the drive, not literally the same drive, so that shouldn't have any affect.

**Comment by galli-leo** [comment]
So all the files are in the same position on the new drive (except that you added the [Quality Title]?

**Comment by jonesb63** [comment]
I did not manually add anything.  The [Quality Title] was added directly when downloading the movie initially from the first radarr.   

After going over the logs, it seems that mediainfo is failing to be run on the file, though it finds the corect .mkv file in the folder, so I think it might be a permission issue, because my google drive is currently API banned for the next couple hours, so I will wait until that's fixed and see if that solves the issue.
